pyart/core/radar.py

```diff
@@ -1019,6 +1019,7 @@ def join_radar(radar1, radar2):
 
     # to combine times we need to reference them to a standard
     # for this we'll use epoch time
+    ### NG THIS should use new updated datetime_utils if accepted
```

Comment:
Typically XXX to TODO are used to mark areas of code where future improvements might be needed.  This makes these areas easy to grep for.